site/en/docs/extensions/reference/offscreen/index.md

```diff
@@ -40,44 +40,50 @@ Use `chrome.offscreen.createDocument()` and `chrome.offscreen.closeDocument()` f
 
 ```js
 chrome.offscreen.createDocument({
-  url: chrome.runtime.getURL('off_screen.html'),
+  url: 'off_screen.html',
   reasons: ['CLIPBOARD'],
   justification: 'reason for needing the document',
 });
 
 chrome.offscreen.closeDocument()
 ```
 
-The following example shows how to check for existing offscreen documents. The `hasOffscreenDocument()` function uses [clients.matchAll()](https://developer.mozilla.org/docs/Web/API/Clients/matchAll) to find existing offscreen documents. Please note: the function assumes that your extension uses a single offscreen document. 
+The following example shows how to ensure that offscreen documents have already been created. The `setupOffscreenDocument()` function uses [clients.matchAll()](https://developer.mozilla.org/docs/Web/API/Clients/matchAll) to find existing offscreen documents. If no existing offscreen documents, create it. Please note: the function assumes that your extension uses a single offscreen document. 
```

Comment:
```suggestion
The following example shows how to ensure that an offscreen document has already been created. The `setupOffscreenDocument()` function calls [`clients.matchAll()`](https://developer.mozilla.org/docs/Web/API/Clients/matchAll) to find the existing offscreen document. If no offscreen document exists, is created. Note that a service worker can only have one offscreen document. 
```

Reply:
I fixed the first item.

I didn't remove the 'Please note...'. I just made it a bit pithier.

Reply:
I appreciate your time and input, but this is problematic. I think you're trying to future proof this, which is a good instinct. But, baseline documentation shouldn't make promises about future product work. If we've done that elsewhere, I need to fix it.

What this line needs to do is say in a straightforward way that the function gets the existing offscreen document or creates it if it doesn't already exist and that it does so using...

In other words, stick to the facts.